ivre/db/mongo.py

```diff
@@ -5040,6 +5039,7 @@ def topvalues(self, flt, fields, collect_fields=None, sum_fields=None,
 
             res_count = ext_entry.pop('_count')
             # Format collected results in a set of tuples to avoid duplicates
+            res_collected = set()
```

Comment:
This is not directly related to this PR, but it fixes an issue which happens when using `topvalues` without collected values: `res_collected` must be initialized even if there are no collected values.

Reply:
You could move that to an `else` after the following `if` statement?